### ISSUE TITLE
Is the framework updated？

### ISSUE THREAD

**Issue by mikedream89** [Odoo-mobile/framework#346]
Is the framework updated？
@dpr-odoo @sla-odoo @mga-odoo

**Comment by kasim1011** [comment]
@mikedream89 
`framework` is not updated with `odoo-11` but, it might work well with `odoo-11`.

https://github.com/Odoo-mobile/framework/issues/344

**Comment by naitikvithlani** [comment]
Hello @mikedream89 @kasim1011 ,

I am using this with odoo 11, It's works fine, I have implemented many of the modules of odoo with this. (Find attached screen shot) 

![image](https://user-images.githubusercontent.com/6597323/31885606-b0879a78-b80e-11e7-89e7-e671f8d1b8b8.png)

**Comment by mikedream89** [comment]
@naitikvithlani  can you  download all data to mobile phone database?

**Comment by naitikvithlani** [comment]
Yes

**Comment by jayminnayak** [comment]
@naitikvithlani, how can I get the odoo 11 models for the mobile framework? is there any source?

**Comment by naitikvithlani** [comment]
No, there no source code, i had update code and added other modules.

**Comment by jayminnayak** [comment]
@naitikvithlani, Thank you!! 
Can you please help to update the module as I've upgraded backend with odoo11 but from the mobile side, it is still odoo 10.
